source/installation/install-on-a-single-machine/install-on-a-single-machine.txt

```diff
@@ -41,6 +41,9 @@ If you are migrating from an Oracle database:
 
 - Download `21.6.0.0 of odjbc11.jar from the Oracle 21c <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`_. 
 
+- (Optional) To migrate tables that contain the ``XML`` type, you must also download the 
```

Comment:
This should say `XMLType` : [XMLType](https://docs.oracle.com/cd/A97630_01/appdev.920/a96620/xdb04cre.htm)

Thanks Jocelyn, this is great!

Reply:
Updated